<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTable</class>
 <widget class="QWidget" name="FormTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>54</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="YuGangBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GuhuaBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BoqianBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BoqianBiaoBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ShuzhiBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ShuzhiBiaoBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>235</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="YonghuBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="JiqirenBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>325</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="JiluBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GuhuaBiaoBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>415</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
